src/clusterfuzz/_internal/base/utils.py

```diff
@@ -241,6 +241,7 @@ def service_account_email():
 
 def get_bot_testcases_file_path(input_directory):
   """Returns path to bot-specific fuzzed testcases."""
+  # !!! do we want this anymore?
```

Comment:
I thought it was used in [multiple places](https://github.com/search?q=repo%3Agoogle%2Fclusterfuzz%20get_bot_testcases_file_path&type=code)?

Reply:
I'm not exactly talking about the function, I'm more talking about the practice of using FUZZ_INPUTS to "prevent high load on nfs servers", since we don't use NFS anymore.

Reply:
@oliverchang WDYT?